CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 4.1.0
+  * Change clients stream to full sync [#111](https://github.com/singer-io/tap-mambu/pull/111)
```

Comment:
can you change it to `full table sync` instead of just `full sync`

Reply:
Okay.